src/index.js

```diff
@@ -98,9 +101,62 @@ function showRules() {
 
 function instantiateGame() {
   let game = new Game([player1, player2, player3]);
+  pickCategories();
   showGame();
 }
 
```

Comment:
In like how your code shows the steps you took to get to the end point, it looks very modular and every function follows single responsibility.